README.rst

```diff
@@ -5,6 +5,25 @@ A sunpy-affiliated package for solar instrument-specific tools.
     :target: http://www.sunpy.org
     :alt: Powered by SunPy Badge
 
+What is sunkit-instruments?
+---------------------------
+
+sunkit-instruments is a sunpy-affiliated package for solar instrument-specific tools.
+Its purpose is not to be a repository for all tools for all instruments.
```

Comment:
Tools refers to anything really?

Reply:
I guess so.  Are there any type of tools we want to explicitly put beyond the scope?  e.g. GUIs?

Reply:
Packaging GUI's is an interesting topic, shouldn't be too hard to do. Question is, do you want GUIs in the codebase?

Reply:
I don't think so.  I think if you want write a GUI, it probably merits its own package.